### ISSUE TITLE
How did you get pseudo ground truth mesh for real-world dataset?

### ISSUE THREAD

**Issue by Elody-07** [3d-hand-shape/hand-graph-cnn#43]
Thanks for your great work!

I notice that you have given the real-world meshes, and the paper didn't make it clear how you get this. Could you please direct me into this?

Thanks in advance for your response~!

**Comment by 3d-hand-shape** [comment]
Hi, thanks for your attention to our research work! As we have described in the paper, we generate the pseudo-ground truth mesh from the pre-trained model as an additional supervision. So for the real-world dataset, we first run our pre-trained model on the real-world data to get the pseudo-ground truth mesh. Then we fine-tune the model with the pseudo-ground truth mesh loss, in which we compare the pseudo-ground truth mesh with the fine-tuned model's mesh to guarantee that the fine-tuned model can still generate meshes with correct shape and smooth surface.
